src/index.html

```diff
@@ -4,9 +4,421 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#home">
+          <img
+            src="images/logo.png"
+            alt="Logo"
+            class="header__logo-img"
+          >
+        </a>
+      </div>
+       
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a href="#" class="nav__link">
              Samsung
            </a>
          </li>
          
          <li class="nav__item">
            <a href="#" class="nav__link">
              Smartphones
            </a>
          </li>
          
          <li class="nav__item">
            <a
              href="#"
              class="nav__link"
              data-qa="nav-hover"
            >
              Laptops & Computers
            </a>
          </li>
          
```
fix the previous comments about code style
if you have any questions, just feel free to ask in `fe_chat`